src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -92,10 +92,14 @@ public ExportPanel(TerminationFuture terminationFuture, ExportForms forms, Expor
           preferences.putAll(configuration.asMap(buildCustomConfPrefix(formId)))
       );
 
-      if (forms.someSelected() && (form.getConfPanel().isValid() && forms.allSelectedFormsHaveConfiguration()))
+      if (forms.someSelected() && (form.getConfPanel().isValid() && forms.allSelectedFormsHaveConfiguration())) {
+        form.setErrorMsg("No errors! Ready to export form(s)");
         form.enableExport();
-      else {
-        //TODO show message to user why export button is disabled
+      } else {
+        if (!forms.someSelected())
+          form.setErrorMsg("No form(s) selected for export");
+        else
+          form.setErrorMsg("Some form(s) need encryption key. You can set this on the general settings above");
```

Comment:
Since the user can decide to set while overriding the general configuration, let's get rid of the last part and leave it "Some form(s) need a decryption key"